environment-api/src/main/java/com/sequenceiq/environment/api/proxy/doc/ProxyConfigDescription.java

```diff
@@ -1,11 +1,11 @@
 package com.sequenceiq.environment.api.proxy.doc;
 
 public class ProxyConfigDescription {
-    public static final String LIST_BY_WORKSPACE = "list proxy configurations for the given workspace";
-    public static final String GET_BY_NAME_IN_WORKSPACE = "get proxy configuration by name in workspace";
-    public static final String CREATE_IN_WORKSPACE = "create proxy configuration in workspace";
-    public static final String DELETE_BY_NAME_IN_WORKSPACE = "delete proxy configuration by name in workspace";
-    public static final String DELETE_MULTIPLE_BY_NAME_IN_WORKSPACE = "delete multiple proxy configurations by name in workspace";
+    public static final String LIST_V1 = "list proxy configurations";
```

Comment:
Why do we need version in the description?

Reply:
Just for the future versioning, but I deleted these V1 postfixes